src/types.ts

```diff
@@ -0,0 +1,13 @@
+import mmdb = require('maxmind');
+
+export interface IMaxMind {
```

Comment:
Should we be following the naming guidelines at https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names? In particular:

> Do not use "I" as a prefix for interface names.